packages/frontend-api/apiary.apib

```diff
@@ -1808,3 +1808,329 @@ Returns customer user new personal data
+                            "transport": {
+                                "name": "Personal collection"
+                            },
+                            "payment": {
+                                "name": "Credit card"
+                            },
+                            "companyName": null,
+                            "companyNumber": null,
+                            "companyTaxNumber": null,
+                            "firstName": "Iva",
+                            "lastName": "Jačková",
+                            "email": "no-reply@shopsys.com",
+                            "street": "Druhá 2",
+                            "city": "Ostrava",
+                            "postcode": "71300",
+                            "country": "CZ",
+                            "differentDeliveryAddress": false,
+                            "note": null
+                        }
+                    },
+                    {
+                        "node": {
+                            "uuid": "f236f5ac-6d0d-4074-a666-7b878e7a6937",
+                            "number": "1595328622",
+                            "urlHash": "fpb0c5Aasmw71CLkltLwZVm3h2efkfjMr3hsRkvgxupjYj31QO",
+                            "creationDate": "2020-07-19T04:02:31+00:00",
+                            "status": "New",
+                            "totalPrice": {
+                                "priceWithVat": "83.580000"
+                            },
+                            "items": [
+                                {
+                                    "name": "CD-R 210MB"
+                                },
+                                {
+                                    "name": "Cash on delivery"
+                                },
+                                {
+                                    "name": "Czech post"
+                                }
+                            ],
+                            "transport": {
+                                "name": "Czech post"
+                            },
+                            "payment": {
+                                "name": "Cash on delivery"
+                            },
+                            "companyName": null,
+                            "companyNumber": null,
+                            "companyTaxNumber": null,
+                            "firstName": "Jan",
+                            "lastName": "Adamovský",
+                            "email": "no-reply@shopsys.com",
+                            "street": "Třetí 3",
+                            "city": "Ostrava",
+                            "postcode": "71200",
+                            "country": "CZ",
+                            "differentDeliveryAddress": false,
+                            "note": null
+                        }
+                    }
+                ]
+            }
+        }
+    }
+
+## Order detail (authorized) [/graphql{?product_detail}]
```

Comment:
```suggestion
## Order detail (authorized) [/graphql{?order_detail_authorized}]
```

Reply:
You might need to update this docs after change of requiring logged user for retrieving order by its uuid

Reply:
This one stayed unchanged. Changes are in unauthorized part.